Makefile

```diff
@@ -35,6 +35,10 @@ update-version:
 	echo "__commit__ = '$(shell git rev-parse HEAD)'" >> calico_cni/__init__.py 
 	echo "__branch__ = '$(shell git rev-parse --abbrev-ref HEAD)'" >> calico_cni/__init__.py
 
+# Copy the plugin into place
+deploy:
```

Comment:
Should this be `deploy-rkt` so it isn't misleading?

Should this depend on `dist/calico`?